setup.yml

```diff
@@ -24,6 +24,7 @@
         libvirt_dev_package: libvirt-dev
         qemu_utils_package: qemu-utils
         nfs_package: nfs-kernel-server
+        golang_package: golang
```

Comment:
`golang_package: golang` also needed in the facts section for fedora

Reply:
done